pest_meta/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "pest_meta"
+version = "0.1.0"
+authors = ["Victor-Nicolae Savu <victor.nicolae.savu@gmail.com>"]
+repository = "https://github.com/pest-parser/pest"
+documentation = "https://docs.rs/pest"
+keywords = ["pest", "parser", "peg", "grammar"]
+categories = ["parsing"]
+license = "MPL-2.0"
+readme = "_README.md"
+
+[dependencies]
+maplit = "^0.1.5"
+pest_derive = "1.0.3"
```

Comment:
pest dependencies here should be `^1.0`.

Reply:
Yup! I suppose 1.1 would already be available by the time this PR lands, so it might make sense to use that instead and release these changes as v 1.2. (same reasoning if 1.2 is released as well before this PR)